src/aiy/audio.py

```diff
@@ -108,15 +110,26 @@ def play_audio(audio_data):
     player.play_bytes(audio_data, sample_width=AUDIO_SAMPLE_SIZE, sample_rate=AUDIO_SAMPLE_RATE_HZ)
 
 
-def say(words, lang=None):
+def say(words, lang=None, volume=None, pitch=None):
     """Says the given words in the given language with Google TTS engine.
 
-    If lang is specified, e.g. "en-US', it will be used to say the given words.
+    If lang is specified, e.g. "en-US", it will be used to say the given words.
     Otherwise, the language from aiy.i18n will be used.
+    volume (optional) volume used to say the given words.
+    pitch (optional) pitch to say the given words.
+    Example: aiy.audio.say('This is an example', lang="en-US", volume=75, pitch=135)
+    Any of the optional variables can be left out.
     """
+    global _tts_volume
```

Comment:
Remove - these aren't required in this function, as it uses get_*_volume.

Reply:
Yeah sorry about that. Didn't do much python programming yet.